src/mapeo-manager.js

```diff
@@ -368,6 +365,7 @@ export class MapeoManager extends TypedEmitter {
       projectId,
       projectPublicId,
       projectKeys: keys,
+      projectInfo: { name: settings.name },
```

Comment:
The project name (and other info about a project) is indexed to the `projectSettingsTable` once a project is synced. Do we need to replicate it here? Maybe... for a project that hasn't synced yet, but I don't think we consider a project joined until we actually complete that first sync?

Reply:
Not sure why I added this! Removed in 6ff40d284c4f271b3efe3416c5e1b4ee770a8c42.